lib/comparing.dart

```diff
@@ -12,6 +12,14 @@ class Comparing {
     return (a, b) => mapper(a).compareTo(mapper(b));
   }
 
+  /// Use if you already have a custom comparator you wish to use but your list needs some mapping to get to the comparator's type.
```

Comment:
Very long line

Reply:
I dont think this file respects any line limit for comments :P 

what is our suggested max so I can wrap everything?

Reply:
I wrapped at 120, I think that is reasonable?

Reply:
I think comments, dartdoc and code should all go at the same line length, which we now have as the old school 80. It becomes easier to read and size containing windows that doesn't wrap the lines.

Reply:
wrapped all files at 80!